components/Storyblok/SbStoryMvp/SbStoryMvp.tsx

```diff
@@ -72,6 +73,7 @@ export const SbStoryMvp = ({
           {!(title?.includes('Whereas') || title?.includes('Progress') || title?.includes('Video') || title?.includes('Solve')) && (
             <StoryHeroMvp
               title={title}
+              superhead={superhead}
```

Comment:
This is an easy update because StoryHeroMvp uses the BlurryPoster component, which already has superhead support